### DISCUSSION TITLE
[HW1] Problem 4.1

### DISCUSSION THREAD

**Discussion by emjay73** [seominjoon/kaist-ai605#79] [Q&A] [answered]
Is there a preferable position for Dropout?
I found various attempts regarding Dropout.
For instance, some apply Dropout at h_{t-1}, others apply Dropout at g_{t}.
What would be the best position for Dropout?

I referred to the following.
http://www.nextgenvis.eu/publicationfiles/C16-1165.pdf

**Comment by seominjoon** [comment, marked as answer]
Dropout at input for RNNs is a common practice, though there are other equally good positions.

**Reply by emjay73** [reply]
Thank you!
